star-sys/build.rs

```diff
@@ -128,6 +129,41 @@ fn main() {
         .flag("-Wall")
         .flag("-Wextra")
         .flag("-Werror")
-        .flag("-fvisibility=hidden")
-        .compile("orbit");
+        .flag("-fvisibility=hidden");
+
+    if cfg!(target_feature = "sse3") {
```

Comment:
Thinking out loud: Are we going to be applying this pattern to other repositories? Would it make sense to put this in a crate for sharing across repos?

Reply:
Unfortunately macros don't share easily.  Really this should be a thing that `cc` just knows how to do itself.  (I guess I should probably check whether it does, now that I think about it)

Reply:
Actually, `cc` does honor `CFLAGS`, so this won't actually do anything in our bazel build.  Still useful when doing a native cargo build, for micro-benchmarks.